### PR TITLE
[DOCS] Add info that at some point we will use `:stable` tagging

### PR DESCRIPTION
**Pull request by garvinhicking**

### CODE REVIEW

File: Documentation/Installation/Index.rst [hunk 1]
```diff
@@ -42,6 +42,9 @@ For all available tags, please check the `GitHub packages page`_.
 Once you have pulled the image, you can run the image to render your project's
 documentation.
 
+Note that we will use `:stable` at a later point to always provide stable
```
Comment (brotkrueml) [inline Documentation/Installation/Index.rst:45]: What about `.. note::`` ;-)

File: Documentation/Migration/Index.rst [hunk 2]
```diff
@@ -20,7 +20,8 @@ and optionally also remove the old :file:`Settings.cfg` file.
 With official Docker container
 ==============================
 
-(Work in progress)
+(Work in progress - Note that we will use `:stable` at a later point to always provide stable
```
Comment (brotkrueml) [inline Documentation/Migration/Index.rst:23]: Here too.

File: README.rst [hunk 3]
```diff
@@ -25,7 +25,10 @@ Usage with Docker (via supplied container)
     # On macOS you need to specify the parameter "--user=$(id -u):$(id -g)"
     docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:main --config=Documentation
 
-(see :ref:`_Setup_Docker:Docker containers` for complete documentation)
+(see :ref:`_Setup_Docker:Docker containers` for complete documentation. Note that eventually instead of
```
Comment (brotkrueml) [inline README.rst:28]: Here too, possibly use `.. seealso::` for the link before.

File: Documentation/Migration/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Migration/Index.rst:23]: I dont think we need to use a note admonition whereever we use the word "note"...

File: Documentation/Installation/Index.rst (hunk 1 above)
Comment (garvinhicking) [reply Documentation/Installation/Index.rst:45]: I still can't write RST format organically :D Damn, you noticed!

(Will add it now that you mentioned the syntax *g*)

File: Documentation/Migration/Index.rst (hunk 2 above)
Comment (garvinhicking) [reply Documentation/Migration/Index.rst]: Please check the rework.

I don't really like that it's at three places now, but that'll be gone one's the notice is gone...

Comment (linawolf) [reply Documentation/Migration/Index.rst]: I think one note should be enough

Comment (brotkrueml) [reply Documentation/Migration/Index.rst]: > I dont think we need to use a note admonition whereever we use the word "note"...

Yup, but this should be visible more prominent.

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**